src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocTestForbiddenFragments.java

```diff
@@ -0,0 +1,19 @@
+/*
+SummaryJavadoc
+violateExecutionOnNonTightHtml = (default)false
+forbiddenSummaryFragments = .*CheckStyle$
+
+
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.summaryjavadoc;
+
+public class InputSummaryJavadocTestForbiddenFragments {
+
+    /** // violation 'Forbidden summary fragment'
```

Comment:
Move violation comment outside of javadoc multiline comment

Reply:
Done